README.md

```diff
@@ -1,3 +1,11 @@
 A simple tool to add/remove tags and elements from/to the `server.xml` file of a Tomcat server.
 
 Call `tomcat-serverxml-tool.sh` with arguments for help
+
+## Example
+```
+./serverxml-tool.sh add-context.xslt docBase=%{tomcatappdir}/host-manager path=/host-manager contextXml=%{tomcatappdir}/host-manager/META-INF/context.xml
```

Comment:
Is it just me, or `%{tomcatappdir}` comes from the `tomcat` SPEC?

What would be `tomcatappdir` in this context? The path to the deployed `.war`?

Reply:
Thanks for noticing it, it should be better now